modules/profile/manifests/basenode.pp

```diff
@@ -1,26 +1,35 @@
 class profile::basenode {
 
   include motd
-  include base
+
+# schedule parameters for update
+  $period        = 'daily'
+  $repeat        = '1'
+
+  class { 'base':
+    period       => $period,
+    repeat       => $repeat,
+  }
```

Comment:
Throw these additional variables out!
class { 'base':
    period       => 'daily,
    repeat       => '1',
}
that's it!